.rubocop_todo.yml

```diff
@@ -1,24 +1,52 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2019-04-13 14:41:36 -0400 using RuboCop version 0.52.1.
+# on 2019-04-15 11:42:23 -0400 using RuboCop version 0.64.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 19
+# Offense count: 1
+# Cop supports --auto-correct.
+Layout/EmptyLineAfterGuardClause:
```

Comment:
I re-generated the config file. The updated Rubocop gems catch some new issues – I'll address these separately later.